docs/content/drafts/Tooltip.mdx

```diff
@@ -0,0 +1,80 @@
+---
+componentId: tooltip
+title: Tooltip
+status: Alpha
+---
+
+import data from '../../../src/drafts/Tooltip/Tooltip.docs.json'
+
+The Tooltip component is used to add context to interactive elements on the page.
+
+**_⚠️ Usage warning! ⚠️_**
+
+Tooltips as a UI pattern should be our last resort for conveying information because it is hidden by default and often with zero or little visual indicator of its existence.
+
+Before adding a tooltip, please consider: Is this information essential and necessary? Can the UI be made clearer? Can the information be shown on the page by default? See [Tooltip alternatives](https://primer.style/design/accessibility/tooltip-alternatives) for more accessible alternatives.
+
+Tooltip should only be used on an element that is interactive such as a button or a link.
+
+## Examples
+
+### Default (As a description type)
```

Comment:
I think we should reword "as description type" to something like "for supplementary description" for the docs and add a small text for when you should reach out for description.

I had no idea what the difference was between type description and label until we started working on Tooltip 😅

Reply:
Good point! I updated the docs to address this and the below [comment](https://github.com/primer/react/pull/3394#discussion_r1325919474) (stole a little bit from https://github.com/primer/design/pull/565 ) - here is the [commit](https://github.com/primer/react/pull/3394/commits/b2d30fbf652c6d5577cca77e897d43ddac493ddc). Let me know what you think!